Automattic/jetpack
# VideoPress: minor ClipboardButtonInput story enhancement

VideoPress: minor ClipboardButtonInput story enhancement

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: minor ClipboardButtonInput story enhancement

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Test with Storybook
* Select the default story of `ClipboardButtonInput`
* Open the Addons panel
* Confirm you see the new `resetTime` property
* Confirm it changes the time to reset the initial state of the component

https://user-images.githubusercontent.com/77539/186441800-9a449795-144e-4cd6-b6e4-e055b25fe191.mov
